google/cloud/pubsub/internal/tracing_batch_callback.cc

```diff
@@ -135,10 +137,9 @@ class TracingBatchCallback : public BatchCallback {
       if (subscribe_span) {
         opentelemetry::trace::StartSpanOptions options;
         options.parent = subscribe_span->GetContext();
-        auto span =
-            internal::MakeSpan("subscriber concurrency control",
-                               {{sc::kMessagingSystem, "gcp_pubsub"}}, options);
-        spans->second.concurrency_control_span = std::move(span);
+        spans->second.concurrency_control_span = std::move(internal::MakeSpan(
```

Comment:
You don't need the `move` now because the argument is already a temporary.

Reply:
Oops. Fixed.